lib/redis/commands/bitmaps.rb

```diff
@@ -27,9 +27,13 @@ def getbit(key, offset)
       # @param [String] key
       # @param [Integer] start start index
       # @param [Integer] stop stop index
+      # @param [String, Symbol] scale the scale of the offset range
+      #     e.g. 'BYTE' - interpreted as a range of bytes, 'BIT' - interpreted as a range of bites
       # @return [Integer] the number of bits set to 1
-      def bitcount(key, start = 0, stop = -1)
-        send_command([:bitcount, key, start, stop])
+      def bitcount(key, start = 0, stop = -1, scale: nil)
+        command = [:bitcount, key, start, stop]
+        command << scale.to_s.upcase if scale
```

Comment:
I wouldn't even bother with the `to_s.upcase`, we can directly pass it to redis in lower case, it's no problem.